go/go.mod

```diff
@@ -95,3 +96,5 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240708141625-4ad9e859172b // indirect
 	google.golang.org/grpc v1.65.0 // indirect
 )
+
+replace github.com/invopop/jsonschema => github.com/alonsopf/jsonschema v0.0.0-20241112034953-8a224be9076f
```

Comment:
Ah, that's no good. We don't want to rely on a fork of this, it's not sustainable for us to maintain it. Is there any way to roll those changes into the Genkit library? Otherwise we need to explore a different solution.

Reply:
I opened [a PR in the jsonschema repo](https://github.com/invopop/jsonschema/pull/157), also I'm reviewing anothers options if doesn't get merged

Reply:
sounds good, I tried to do in this PR using our own codebase https://github.com/firebase/genkit/pull/1400